speedcord/exceptions.pyi

```diff
@@ -0,0 +1,55 @@
+from aiohttp import ClientResponse
+
+
+class HTTPException(Exception):
+    request: ClientResponse
+    data: str
+
+    def __init__(self, request: ClientResponse, data: str) -> None:
```

Comment:
Pointing out that `__init__` returns none seems kinda pointless and is probably wrong aswell. This applies to all instances where you did this

Reply:
I was running `mypy` and `pyright` in strict mode, which raised `Return type is unknown` as an error - I'll remove all the explicit `-> None` annotations.